pages/about/index.vue

```diff
@@ -1,106 +1,73 @@
-            </template>
-        </i18n>
-        <i18n path="paragraph.year2012And2013" tag="p">
-            <template #linkEverybodyPays>
-                <ext-link :href="linkEverybodyPays" highlight
-                    >Everybody Pays</ext-link
+            <template #text>
+                <core-h1 :title="$t('pageTitle')"></core-h1>
+                <p
+                    v-for="(content, i) in $t('pageAbstract')"
+                    :key="`abstract_${i}`"
+                    class="w-full md:w-3/4 lg:w-3/4"
                 >
+                    {{ content }}
+                </p>
             </template>
-            <template #linkYycBlog2012>
-                <ext-link :href="linkYycBlog2012" highlight>{{
-                    $t('link.yycBlog2012')
-                }}</ext-link>
-            </template>
-            <template #linkYycBlog2013>
-                <ext-link :href="linkYycBlog2013" highlight>{{
-                    $t('link.yycBlog2013')
-                }}</ext-link>
-            </template>
-        </i18n>
-        <i18n
-            v-if="$t('paragraph.year2013')"
-            path="paragraph.year2013"
-            tag="p"
-            class="paragraph"
-        >
-            <template #linkYycBlog2013>
-                <ext-link :href="linkYycBlog2013" highlight>{{
-                    $t('link.yycBlog2013')
-                }}</ext-link>
-            </template>
-        </i18n>
-        <i18n path="paragraph.year2014And2015" tag="p">
-            <template #linkCheckIo>
-                <ext-link :href="linkCheckIo" highlight>CheckIO</ext-link>
-            </template>
-        </i18n>
-        <i18n path="paragraph.pyDay2015" tag="p"></i18n>
-        <i18n path="paragraph.year2016" tag="p">
-            <template #linkPyConBlog2016>
-                <ext-link :href="linkPyConBlog2016" highlight>{{
-                    $t('link.pyConBlog2016')
-                }}</ext-link>
-            </template>
-            <template #linkPhotoAlbum>
-                <ext-link :href="linkPhotoAlbum" highlight>{{
-                    $t('link.photoAlbum')
-                }}</ext-link>
-            </template>
-        </i18n>
-        <i18n path="paragraph.year2017" tag="p"></i18n>
-        <i18n path="paragraph.year2018" tag="p"></i18n>
-        <i18n path="paragraph.year2019" tag="p"></i18n>
-        <i18n path="paragraph.year2020" tag="p"></i18n>
-        <i18n path="paragraph.year2021" tag="p"></i18n>
-        <i18n path="paragraph.outro" tag="p"></i18n>
-    </i18n-page-wrapper>
+        </Banner>
+        <div class="flex flex-wrap justify-center mx-2 my-3">
+            <div v-for="(img, i) in introImgs" :key="`intro_img_${i}`">
+                <div
+                    class="w-32 md:w-56 lg:w-56 h-24 md:h-40 lg:h-40 bg-contain bg-no-repeat mx-2 my-3"
+                    :style="{
+                        'background-image': 'url(' + img + ')',
+                    }"
```

Comment:
Could we use a method like `getXXStyle(img)` instead of using the inline style? Or at least apply  string template to the style?